changes/177.breaking

```diff
@@ -0,0 +1 @@
+Refactor other files in the CLI to match the new, more consistent format
```

Comment:
Just a note: In the case of PR of PR, we don't need to put the news fragment because the original PR will cover it.